sdk/lusid/utilities/api_client_builder.py

```diff
@@ -58,11 +58,27 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
                 if value is not None:
                     setattr(configuration, key, value)
 
-        # Use the access token provided if it exists
+        #---------------------------------------------------------------------
+        # The ApiClientBuilder uses the following conditional logic to resolve
+        # an authentication approach:
+        # (1) First look for a PAT passed in as a param by the user
+        # (2) Next look for a secrets file passed in as a param by the user
```

Comment:
i personally find this a bit unclear, can we simplify 

1) Use PAT passed into the function
2) Use secrets files passed into the function
3) Use environment variables

PAT are used in preference to supplied credentials
secrets files are used in preference to environment variables

Reply:
Done.